Source/JoystickPlugin/Public/Widgets/SJoystickInputSelector.cpp

```diff
@@ -83,6 +84,14 @@ FReply SJoystickInputSelector::OnKeyUp(const FGeometry& MyGeometry, const FKeyEv
 
 FReply SJoystickInputSelector::OnAnalogValueChanged(const FGeometry& MyGeometry, const FAnalogInputEvent& InAnalogInputEvent)
 {
+	/** Don't process events in dead zone */
+	const float AbsAnalogValue = FMath::Abs(InAnalogInputEvent.GetAnalogValue());
+
+	if (AbsAnalogValue <= DeadZone || AbsAnalogValue >= 1.0f-DeadZone)
```

Comment:
I feel this should be an early out
So return a `FReply::Handled` if the value is less than or equal to deadzones
ie. 
```cpp
	if (AbsAnalogValue <= DeadZone)
	{
		return FReply::Handled();
	}
```

This should then let the rest of the OnAnalogValueChanged function behave as expected